.github/workflows/ci.yml

```diff
@@ -20,13 +20,13 @@ jobs:
     steps:
 
     - name: Cancel Previous Runs
-      uses: styfle/cancel-workflow-action@0.9.1
+      uses: styfle/cancel-workflow-action@a40b8845c0683271d9f53dfcb887a7e181d3918b # 0.9.1
```

Comment:
Would @dependabot still be able to detect newer versions and create pull requests like https://github.com/libp2p/rust-libp2p/pull/2300?

Reply:
https://github.com/dependabot/dependabot-core/issues/2835

It seems it won't have a problem with it. It won't update the version number in the comment though :( We could use https://github.com/timmeinerzhagen/dependabot-sha-comment-action for that if we wanted to.

Reply:
:rocket: I don't mind updating the version comment manually.

Reply:
Alright :) Let me know if it becomes annoying. Hopefully, we'll be able to move the workflow to unified CI repo before that happens and then we'll be handling the action upgrades there.